p2p/protocols/accounting.go

```diff
@@ -114,18 +116,18 @@ func NewAccounting(balance Balance, po Prices) *Accounting {
 //at the passed interval writes the metrics to a LevelDB
```

Comment:
I believe that `godoc` convention is to have `// ...` (//<space><funcName>...)

Reply:
It doesn't look like it's explicit, but they themselves always DO use a space....

I don't really like it but if that's the convention I will apply from now on

Reply:
i believe this is the convention across the codebase too